databuilder/extractor/minio_extractor.py

```diff
@@ -51,7 +51,7 @@ def extract(self) -> Any:
                 Bucket=self.bucket_name,
                 Key=name,
                 ExpressionType='SQL',
-                Expression="select * from s3object limit 0",
+                Expression="select * from s3object limit 1",
```

Comment:
@we-z looks like this did need to be at least 1 -- fails if it's zero :(

Reply:
Yup